### ISSUE TITLE
streamers (YouTube/Twitch)

### ISSUE THREAD

**Issue by JackMcLeanGSY** [daniluk4000/vatsim-radar#94]
### Your Vatsim CID

1530091

### What do you suggest?

If a Twitch/YouTube streamers are flying on vatsim hint hint they would show on the map in a different colour.

Would be good for pilots to see streamers who are flying. 

### Why do you need that?

To help content creators and support the project.

### Validation

- [X] I have searched for other requests and didn't found any alike
- [X] I have also checked roadmap, and this is not planned

**Comment by catchingstrays** [comment]
Just to add on - there are air traffic controllers who also stream, maybe some way to depict that too :)

**Comment by JackMcLeanGSY** [comment]
> Just to add on - there are air traffic controllers who also stream, maybe some way to depict that too :)

Yeah, my favourite streams to watch :)

**Comment by DrezzyJ** [comment]
Dropping a line to actually suggest this NOT be implemented. This would invite people to "tag along" the flight by flying the same route, and these people usually spend more time watching the stream rather than flying their aircraft. Also, could overload an already stretched out ATC controller to the point that they log-off.

**Comment by catchingstrays** [comment]
> Dropping a line to actually suggest this NOT be implemented. This would invite people to "tag along" the flight by flying the same route, and these people usually spend more time watching the stream rather than flying their aircraft. Also, could overload an already stretched out ATC controller to the point that they log-off.

Hey, this is a reasonable concern. However, in my honest opinion, users that fly on VATSIM are expected to follow a set of rules that involve keeping their aircraft attended. As an Air Traffic Controller myself in the Caribbean region, we actually like to handle group flights as it realistically gives us more to deal with in terms of speed/altitude restrictions, vectoring people around and whatnot. Single aircraft operations once a while can get a little boring :(

Most streamers do post their route and callsign anyways, so tag-alongs happen without the use of flight tracking applications, and it is common for air traffic controllers to actually sign on to give coverage to those controllers, it is much more fun and enjoyable! After all, supervisors will deal with disruptive planes!

**Comment by daniluk4000** [comment]
I agree. For now it's absolutely ok in SimAware for partner streams, and some controllers also include twitch in their atis. My only concern is API quota usage, otherwise I would have already done that, so for now it's investigating.
